From 7cf1d66c4b01c40b45b8a38370c7ffee46dfb10a Mon Sep 17 00:00:00 2001
From: Lei Maohui <leimaohui@cn.fujitsu.com>
Date: Fri, 31 Jul 2015 03:17:07 +0900
Subject: [PATCH] to fix build error

The error likes as following

| Generating internals/command.html.tmp
| /bin/sh: line 3: internals/command.html.tmp: No such file or directory
| rm: Generating internals/locking.html.tmp
| cannot remove `internals/command.html.tmp': No such file or directory
| make[3]: *** [internals/command.html.tmp] Error 1
| make[3]: *** Waiting for unfinished jobs....

Signed-off-by: Lei Maohui <leimaohui@cn.fujitsu.com>
[ywei: rebased to libvirt-1.3.2]
Signed-off-by: Yunguo Wei <yunguo.wei@windriver.com>
[MA: rebase to v4.3.0]
Signed-off-by: Mark Asselstine <mark.asselstine@windriver.com>

---
 docs/Makefile.am | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/docs/Makefile.am b/docs/Makefile.am
index 61862c4..c259535 100644
--- a/docs/Makefile.am
+++ b/docs/Makefile.am
@@ -360,7 +360,7 @@ EXTRA_DIST= \
   aclperms.htmlinc \
   $(schema_DATA)
 
-acl_generated = aclperms.htmlinc
+acl.html:: $(srcdir)/aclperms.htmlinc
 
 aclperms.htmlinc: $(top_srcdir)/src/access/viraccessperm.h \
         $(top_srcdir)/scripts/genaclperms.py Makefile.am
@@ -426,8 +426,7 @@ manpages/%.html.in: manpages/%.rst
 	$(AM_V_GEN)$(MKDIR_P) `dirname $@` && \
 	  $(RST2HTML) --strict $< > $@ || { rm $@ && exit 1; }
 
-%.html.tmp: %.html.in site.xsl subsite.xsl page.xsl \
-		$(acl_generated)
+%.html.tmp: %.html.in site.xsl subsite.xsl page.xsl
 	$(AM_V_GEN)name=`echo $@ | sed -e 's/.tmp//'`; \
 	  dir=`dirname $@` ; \
 	  if test "$$dir" = "."; \
